lib/Sabberworm/CSS/Value/Value.php

```diff
@@ -55,6 +55,9 @@ public static function parseValue(ParserState $oParserState, $aListDelimiters =
 				array_splice($aStack, $iStartPosition - 1, $iLength * 2 - 1, array($oList));
 			}
 		}
+		if (!isset($aStack[0])) {
+			return null;
```

Comment:
Could you try to instead to throw a custom `UnexpectedTokenException` which, in lenient mode, should be [caught](https://github.com/sabberworm/PHP-CSS-Parser/blob/master/lib/Sabberworm/CSS/RuleSet/RuleSet.php#L35) already.